nmtwizard/preprocess/preprocess.py

```diff
@@ -468,7 +473,8 @@ def get(self, override_label=None):
         """Returns the shared state for this configuration and corpus label."""
         if isinstance(override_label, dict):
             return None
-        cached_state = self._cached_state.get(override_label)
+        override_label_str = repr(override_label)
```

Comment:
Not sure about this, but I don't know how to do it better.

Reply:
This should work.